resources/build.js

```diff
@@ -0,0 +1,45 @@
+var execSync = require('child_process').execSync;
```

Comment:
It's a good idea. I'll definitely do that on personal projects moving forward, but am unsure it's best here.